notebooks/406-3D-pose-estimation-webcam/README.md

```diff
@@ -0,0 +1,23 @@
+# 3D Human Pose Estimation with OpenVINO 
+
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/spencergotowork/openvino_notebooks.git/master?labpath=notebooks%2F224-human-pose-estimation-3d%2F)
+*Binder is a free service where the webcam will not work, and performance on the video will not be good. For best performance, we recommend installing the notebooks locally.*
+
+![pose estimation_webgl](https://user-images.githubusercontent.com/42672437/183292131-576cc05a-a724-472c-8dc9-f6bc092190bf.gif)
+![pose estimation_opencv](https://user-images.githubusercontent.com/42672437/183285240-4ac00639-ceba-4b65-a783-be66a372ac8e.gif)
+
+This demo contains 3D multi-person pose estimation demo. Intel OpenVINO™ backend can be used for fast inference on CPU. This demo is based on [Lightweight OpenPose](https://arxiv.org/abs/1811.12004) and [Single-Shot Multi-Person 3D Pose Estimation From Monocular RGB](https://arxiv.org/abs/1712.03453) papers. It detects 2D coordinates of up to 18 types of keypoints: ears, eyes, nose, neck, shoulders, elbows, wrists, hips, knees, and ankles, as well as their 3D coordinates.
+
+## Notebook Contents
+
+This notebook uses the model to estimate 3D human pose and draw them in 2D screen. The input source can be video files or webcam.
+
+This folder has 2 notebooks, the [406-3D-pose-estimation notebook](https://github.com/spencergotowork/openvino_notebooks/blob/master/notebooks/406-3D-pose-estimation-webcam/406-3D-pose-estimation.ipynb) displays 3D results to the browser using [Three.js python api](https://pythreejs.readthedocs.io/en/stable/installing.html). The [406-opencv-3D-pose-estimation notebook](https://github.com/spencergotowork/openvino_notebooks/blob/master/notebooks/406-3D-pose-estimation-webcam/406-opencv-3D-pose-estimation.ipynb) uses the OpenCV toolkit to map and save the 3D result into a 2D image and display it in the notebook
```

Comment:
Could you help to design a table to list each notebook's name, introduction and gif example ? to make it more clear for our users.
Like what we did in top README file:
https://github.com/openvinotoolkit/openvino_notebooks/edit/main/README.md